microprofile-metrics/src/main/java/org/wildfly/quickstart/microprofile/metrics/PrimeNumberChecker.java

```diff
@@ -0,0 +1,128 @@
+package org.wildfly.quickstart.microprofile.metrics;
+
+import org.eclipse.microprofile.metrics.Counter;
+import org.eclipse.microprofile.metrics.Metadata;
+import org.eclipse.microprofile.metrics.MetricRegistry;
+import org.eclipse.microprofile.metrics.MetricUnits;
+import org.eclipse.microprofile.metrics.annotation.ConcurrentGauge;
+import org.eclipse.microprofile.metrics.annotation.Counted;
+import org.eclipse.microprofile.metrics.annotation.Gauge;
+import org.eclipse.microprofile.metrics.annotation.Metered;
+import org.eclipse.microprofile.metrics.annotation.Metric;
+import org.eclipse.microprofile.metrics.annotation.RegistryType;
+import org.eclipse.microprofile.metrics.annotation.Timed;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import java.util.concurrent.CountDownLatch;
+
+@Path("/")
+@ApplicationScoped
+public class PrimeNumberChecker {
+
+    private long highestPrimeNumberSoFar = 2;
+
+    @GET
+    @Path("/prime/{number}")
+    @Produces(MediaType.TEXT_PLAIN)
```

Comment:
Could the annotation `@Produces` ,for the sake of consistency, be added to rest of the methods registered at REST endpoints producing text output, e.g `duplicates`, `duplicates2`,`injectedMetric`